### DISCUSSION TITLE
v6 docs / tutorial / examples

### DISCUSSION THREAD

**Discussion by svdwyer** [fabricjs/fabric.js#9176] [Q&A]
Quick question: I see that v6 is in beta, but I couldn't find any info on how to get started using v6
Is there such a resource?

**Comment by ShaMan123** [comment]
we are working on the [website](https://github.com/fabricjs/fabricjs.github.io/)
contributions are welcome
For now #8299 and [core concepts](https://github.com/fabricjs/fabricjs.github.io/blob/development/src/content/docs/core-concepts.md)

**Reply by ShaMan123** [reply]
[ths part](https://github.com/fabricjs/fabricjs.github.io/blob/development/src/content/docs/core-concepts.md#customization-subclassing--mutation) in particular
